core/src/main/java/org/mule/runtime/core/transaction/MuleTransactionConfig.java

```diff
@@ -221,7 +220,7 @@ public ConstraintFilter getConstraint()
         }
         catch (CloneNotSupportedException e)
         {
-            logger.fatal("Failed to clone ConstraintFilter: " + e.getMessage(), e);
+            logger.error("Failed to clone ConstraintFilter: " + e.getMessage(), e);
```

Comment:
Don't know if we are using >= 1.6.0 but if we do, we could use the formatting support when logging exceptions too:
http://slf4j.org/faq.html#paramException
```
String s = "Hello world";
try {
  Integer i = Integer.valueOf(s);
} catch (NumberFormatException e) {
  logger.error("Failed to format {}", s, e);
}
```